scripts/chief_deploy.py

```diff
@@ -51,6 +51,7 @@ def update_product_details(ctx):
 @task
 def update_locales(ctx):
     with ctx.lcd(os.path.join(settings.SRC_DIR, 'locale')):
+        ctx.local("dennis-cmd lint --errorsonly .")
```

Comment:
I think so. Here's [a recent stage build log](http://developeradm.private.scl3.mozilla.com/chief/developer.stage/logs/3b6adc1d9e543d9aa575db8cb6dd6d64cedf47d0.1462472097), where the build stopped because ``polib`` wasn't installed. I think a similar build stop would happen if a non-zero exit code was returned.